samples/bluetooth/mesh/light_ctrl/src/model_handler.c

```diff
@@ -244,6 +275,12 @@ void model_handler_start(void)
 {
 	int err;
 
+#if IS_ENABLED(CONFIG_BT_MESH_COMP_PAGE_2)
```

Comment:
Also I suggest this:

```suggestion
#if IS_ENABLED(CONFIG_BT_MESH_NLC_PERF_CONF)
```

Since, this is required if NLC profiles are supported, it seems more logical to use profile specific KConfig option (which would in-turn, enable COMP_PAGE_2 option).

Reply:
But at the same time, if someone want to use CONFIG_BT_MESH_COMP_PAGE_2 without CONFIG_BT_MESH_NLC_PERF_CONF. This will then be left out. That means the user have to use NLC not only comp page 2. That is why I would like to keep it as is.

Reply:
I thought so at first as well. But, without NLC_PERF_CONF they wouldn't be profile compliant, and then the _should_ have to define the CDP2 contents manually (because the code here would incorrectly claim NLC compliance otherwise)

Reply:
No, they will have to change the code then. They can't have this content that is inside the #if (specifying the NLC profiles) in the CDP2. Because then the device would claim to be NLC compliant (when other nodes ask for CDP2), but wouldn't actually be compliant. So then it's fine if this is #if-ed out, because they either way need to set up their own CDP contents.

Reply:
Also, the only purpose of CDP2 (at least currently) is to claim that you support one or more NLC profiles. It makes no sense to use CDP2 but not turn on NLC compliance.